me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/IDmeWebVerify.java

```diff
@@ -204,34 +200,49 @@ private String ExtractAccessToken(String url)
      */
     private void GetWebProfile(String url)
     {
+        String serverResponse;
+        HttpURLConnection urlConnection;
+        URL urlRequest;
+        try {
+            urlRequest = new URL(url);
+            urlConnection = (HttpURLConnection) urlRequest.openConnection();
```

Comment:
I see that the connection is never closed. Is this on purpose? It should be closed at some point.